torch/_dynamo/variables/misc.py

```diff
@@ -635,6 +635,29 @@ def call_method(
         return super().call_method(tx, name, args, kwargs)
 
 
+class MethodWrapperVariable(variables.ConstantVariable):
```

Comment:
Why do these subclass constant variable?

Reply:
In one of the other comments Ed and I discussed - these are constants in the same way functions are constants, there's one global name with a unique ID.

Reply:
As Ed alluded to below, I'm going to undo this and just create a specific VT